languages/csharp/exercises/concept/randomness/.docs/instructions.md

```diff
@@ -0,0 +1,23 @@
+Continuing the theme of the wizards and warriors game, it is time to add an all purpose die rolling method. This will be the traditional 18 sided die with numbers 1 to 18. Players also generate a spell strength.
+
+## 1. Enable a wizards and warriors player to roll a die.
+
+Implement a `RollDie()` method on the `Player` class.
+
+```csharp
+var player = new Player();
+player.RollDie();
+// => >= 1 <= 18
+```
+
+## 2. Players need their strength. Provide a means to generate spell strength
```

Comment:
The two methods (roll die and generate spell strength) seem to relatively unrelated to each other. I'm wondering if we can combine them into a more coherent narrative. 

GenerateSpellStregngth does not seem to make a ton of sense in the domain of D&D, but I also don't have a great idea here. Maybe @SleeplessByte has an idea on how we can solve this?

Reply:
I'm happy to put this in an improvement issue if we can't resolve it at present.

Reply:
Raised improvement issue #1749

Reply:
Sorry missed this! Feel free to ping on slack when I'm not responsive. I'm pretty good in managing my time :)

Reply:
:)

Reply:
@SleeplessByte There is actually a story-improvement issue, #1749.  We would appreciate any input. The C# exercises may be forked by others.  At least that was @ErikSchierboom 's hope before I got involved.